<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>saveForm</class>
 <widget class="QWidget" name="saveForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_return">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>180</y>
     <width>130</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>退出/Return</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>441</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>地震前兆二氧化碳在线分析仪——仪器参数</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>0</y>
      <width>221</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>ID：X91XDQYQ1402</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>621</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>数据刷新率选择</string>
   </property>
   <widget class="QRadioButton" name="radioButton_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton::indicator{width: 25px;
height: 25px;}</string>
    </property>
    <property name="text">
     <string>1Hz/1s</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_30">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>162</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton::indicator{width: 25px;
height: 25px;}</string>
    </property>
    <property name="text">
     <string>0.03Hz/30s</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_60">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>162</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton::indicator{width: 25px;
height: 25px;}</string>
    </property>
    <property name="text">
     <string>0.017Hz/60s</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_120">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton::indicator{width: 25px;
height: 25px;}</string>
    </property>
    <property name="text">
     <string>0.008Hz/120s</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>621</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>仪器硬件信息设置</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>二氧化碳传感器地址:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>温  湿  度传感器地址:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>吸气泵流量设置:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_CO2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>301</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Tem">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>301</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_SuctionPump">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>134</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox::indicator {
width: 30px;
height: 30px;
}</string>
    </property>
    <property name="text">
     <string>开吸气泵</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_SuctionPump">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>101</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>100</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>L/min</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_CO2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>91</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Tem">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>60</y>
      <width>91</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_SuctionPump">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>100</y>
      <width>91</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_save">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>110</y>
     <width>130</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>保存/Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_keyboard">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>40</y>
     <width>130</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>软件盘/KeyB</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>781</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>参数填写说明</string>
   </property>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>581</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>1.数据刷新率是指读取传感器的速率与界面数据显示的速率。</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>571</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>2.传感器地址是8个字节的16进制数。</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>531</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>3.吸气泵开启以后，可以手动设置吸气泵的流量。</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>211</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
